server/services/RunService.js

```diff
@@ -23,6 +24,43 @@ const db = require('../models/index');
  * @property {string} run_status
  * @property {number} test_version_id
  * @property {Array.<number>} testers - user_id of assigned testers
+ *
+ * @typedef TestSuiteVersion
+ * @type {object}
+ * @property {number} id
+ * @property {string} git_repo
+ * @property {string} git_tag
+ * @property {string} git_hash
+ * @property {string} git_commit_msg
+ * @property {Object.Date} date
+ * @property {Array.<At>}          supported_ats
+ * @property {Array.<ApgExamples>} apg_examples
```

Comment:
This object should be there same as this object returned by "getAllTestVersion", see documentation of that function:  https://github.com/w3c/aria-at-app/blob/28e80d62c43e612e064ee22bedec1bc1230e89d1/server/services/CycleService.js#L224

Reply:
This whole file will be deleted, and the two functions I propose here replace that one. The first will get the current "test version" and run configuration information -- which is used various places in the app, and the second end point is just for the "Test Run Configuration" page.